### DISCUSSION TITLE
2023 Review - Old Memories and New Wishes

### DISCUSSION THREAD

**Discussion by giscus** [Xuanwo/blog#419] [Announcements]
# 2023 Review - Old Memories and New Wishes

An infrastructure engineer, focused on distributed storage system

https://xuanwo.io/2023/05-2023-review/

**Comment by AlexandreHGuerra** [comment]
My best wishes for you. Keep photos of everything and i hope you recover your memory
Im looking after mine, which is not in a great shape...
Do you snore when sleeping? Maybe its a apnea issue, it may affect your memory. 
https://www.mayoclinic.org/diseases-conditions/obstructive-sleep-apnea/symptoms-causes/syc-20352090

**Comment by rgglez** [comment]
Maybe long COVID? I'm not a medic but fatigue, brain fog and so on match long COVID symptoms:

https://www.yalemedicine.org/news/how-to-manage-long-covid-brain-fog

BTW. I found the go-storage library, but it seems abandoned now. Do you know if someone took over the project in a fork?

**Reply by Xuanwo** [reply]
> Maybe long COVID? I'm not a medic but fatigue, brain fog and so on match long COVID symptoms:

Thanks for the sharing.

> Do you know if someone took over the project in a fork?

Sorry, but not yet.

**Reply by rgglez** [reply]
Too bad. That library is ideal for making multi-cloud programs. I don't know if this is the right place to ask this: I've added one or two hacks to my fork, but I am really confused about the code generation (the generated*.* files) and the sub-modules. For this last, I managed to generate a new version using just git, but it is tedious. But I'm nearly loss with the code generation. Do you have some manual on these topics? I would greately appreciate you help. Thanks a lot =)
